python/dev/Dockerfile

```diff
@@ -39,7 +39,7 @@ WORKDIR ${SPARK_HOME}
 ENV SPARK_VERSION=3.4.1
 ENV ICEBERG_SPARK_RUNTIME_VERSION=3.4_2.12
 ENV ICEBERG_VERSION=1.3.1
-ENV AWS_SDK_VERSION=2.20.18
+ENV AWS_SDK_VERSION=2.20.131
```

Comment:
I don't think we can update this, because that's for integration tests that run with Iceberg 1.3.1 (which still requires 2.20.18)

Reply:
Thank you @nastra. Reverted the change.